pageserver/src/tenant.rs

```diff
@@ -3921,6 +3907,8 @@ pub(crate) mod harness {
         }
     }
 
+    #[cfg(test)]
```

Comment:
Changes from this point on:

- reorder from C to more readable topological
- clean up copypaste around spans with Debug on LoadMode
- mark test only things as `#[cfg(test)]` -- I was surprised loading was "TenantHarness loading"

Reply:
`#[cfg(test)]` was not needed, I had misunderstood these were in harness module already which is `#[cfg(test)]`.